niap-cc/Permissions/Tester/app/src/main/java/com/android/certifications/niap/permissions/config/InstallPermissionOnlyConfiguration.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2020 The Android Open Source Project
```

Comment:
Ultra-nit: similar to `DevicePolicyConfiguration.java`, the copyright year should be set for the year this file was created.